.gitignore

```diff
@@ -43,6 +43,7 @@ nbproject/*
 .coverage
 testing/*
 htmlcov/*
+rmgpy/test_data/Cantherm/tst1/*
```

Comment:
It might be better to have a `tearDown` method that cleans these files instead of adding them to gitignore